### PR TITLE
Add meta string to events

### PR DESCRIPTION
**Pull request by pd0wm**
@adeebshihadeh is knowing this in FindInteresting worth the complexity?

### CODE REVIEW

File: selfdrive/controls/lib/events.py [hunk 1]
```diff
@@ -73,31 +74,37 @@ def create_alerts(self, event_types: List[str], callback_args=None):
...
+          if DT_CTRL * (self.events_t[e] + 1) >= alert.creation_delay:
             alert.alert_type = f"{EVENT_NAME[e]}/{et}"
             alert.event_type = et
             ret.append(alert)
     return ret
 
   def add_from_msg(self, events):
     for e in events:
-      self.events.append(e.name.raw)
+      meta = e.meta if len(e.meta) else None
```
Comment (pd0wm) [inline selfdrive/controls/lib/events.py:93]:
```suggestion
      meta = e.meta if e._has('meta') else None
```
Not sure if using this hidden method is cleaner

File: selfdrive/controls/controlsd.py [hunk 2]
```diff
@@ -274,11 +275,15 @@ def update_events(self, CS):
...
       self.events.add(EventName.usbError)
     elif not self.sm.all_alive_and_valid() or self.can_rcv_error:
-      self.events.add(EventName.commIssue)
+      meta = {
+        'invalid': [s for s, valid in self.sm.valid.items() if not valid],
+        'not_alive': [s for s, alive in self.sm.alive.items() if not alive],
+        'can_rcv_error': self.can_rcv_error,
+      }
+
+      self.events.add(EventName.commIssue, meta=json.dumps(meta))
```
Comment (pd0wm) [inline selfdrive/controls/controlsd.py:284]: move json.dumps  to `Events.add()` if it's not a string already?

**Comment by adeebshihadeh** [comment]
I don't think this is worth it. These events should infrequent, and you'll want to look into the logs anyway.

**Comment by pd0wm** [comment]
ok! Let's see if we can get some critical logs into the qlogs then?

**Pull request closed without merging**